data/raw/iam/readme.md

```diff
@@ -21,10 +21,24 @@ ls -1 *.png | parallel --eta -j 6 convert '{}' -adaptive-resize 50% '../forms-re
 
 ## Split
 
-The data split we will use is
-IAM lines Large Writer Independent Text Line Recognition Task (lwitlrt): 9,862 text lines.
-
-- The validation set has been merged into the train set.
-- The train set has 7,101 lines from 326 writers.
-- The test set has 1,861 lines from 128 writers.
-- The text lines of all data sets are mutually exclusive, thus each writer has contributed to one set only.
+The data split we will use is loosely based on the IAM lines Large Writer Independent Text Line Recognition Task (`lwitlrt`) which provides 4 data splits:
+ - Train: has 6,161 text lines from 747 pages written by 283 writers
+ - Validation 1: has 900 text lines from 105 pages written by 46 writers
+ - Validation 2: has 940 text lines from 115 pages written by 43 writers
+ - Test: has 1,861 text lines from 232 pages written by 128 writers
+Total: has 9,862 text lines from 1199 pages written by 500 writers
+The text lines of all data sets are mutually exclusive, thus each writer has contributed to one set only.
+
+The total text lines (9,862) in the data splits is way less then all the text lines (13,353) in the dataset. This is because:
+ - pages of 157 writers (`657-500`) are not included in the data splits
+ - 511 text lines are dropped from the 1,199 pages included in the data splits
+
+To avoid missing out on all the dropped data, we slightly modify the data splits. We:
```

Comment:
Is it clear why those writers and lines were dropped? They could have data quality issues.

Reply:
that's an excellent question. I was hoping to find some answers online but I didn't! I'm assuming there must be some reason. Though for now, I say we just all lines!